editor/export_template_manager.cpp

```diff
@@ -195,7 +195,7 @@ void ExportTemplateManager::_install_from_file(const String &p_file) {
 			data.resize(info.uncompressed_size);
 
 			//read
-			ret = unzOpenCurrentFile(pkg);
+			unzOpenCurrentFile(pkg);
```

Comment:
I'm not sure, I guess it's good how you changed it. If we don't care about the return value, let's not assign it. If we're bothered by that other warning, we can disable it or start caring and check the return value (I guess we should do the latter :)).